docs/learn-glossary.md

```diff
@@ -3,160 +3,175 @@ id: learn-glossary
-or Namespace, WorkflowId, ActivityId arguments.
+A Task Token is a unique correlation ID for a Temporal [Activity](#activity).
+
+- [Activity](#activity) completion calls take a Task Token, [Namespace](#namespace), [Workflow ID](#workflow-id), or [Activity ID](#activity-id) as an argument.
 
 ### Worker
 
-Also known as a _worker service_. A service that hosts the workflow and
-activity implementations. The worker polls the Temporal service for tasks, performs
-those tasks, and communicates task execution results back to the Temporal service.
-Worker services are developed, deployed, and operated by Temporal customers.
+A Worker, also known as a Worker service, is a service that hosts the [Workflow](#workflow) and [Activity](#activity) implementations.
+
+- The Worker polls the Temporal service for [Tasks](#task), performs those [Tasks](#task), and communicates [Task](#task) execution results back to the Temporal service.
+- Worker services are developed, deployed, and operated by Temporal customers.
 
 ### Workflow
 
-A fault-oblivious stateful function that orchestrates activities. A _Workflow_ has full control over
-which activities are executed, and in which order. A _Workflow_ must not affect
-the external world directly, only through activities. What makes workflow code
-a _Workflow_ is that its state is preserved by Temporal. Therefore any failure
-of a worker process that hosts the workflow code does not affect the workflow
-execution. The _Workflow_ continues as if these failures did not happen. At the
-same time, activities can fail any moment for any reason. Because workflow code
-is fully fault-oblivious, it is guaranteed to get notifications about activity
-failures or timeouts and act accordingly. There is no limit on potential workflow
-duration.
+A fault-oblivious stateful function that orchestrates activities.
+
+- A Workflow has full control over which [Activities](#activity) are executed, and in which order.
+- A Workflow must not affect the external world directly, only through [Activities]((#activity)).
+- What makes Workflow code a Workflow is that its state is preserved by Temporal. Therefore any failure of a [Worker](#worker) process that hosts the Workflow code does not affect the [Workflow Execution](#workflow-execution). The Workflow continues as if these failures did not happen. At the same time, [Activities](#activity) can fail any moment for any reason.
+- Because Workflow code is fully fault-oblivious, it is guaranteed to get notifications about [Activity](#activity) failures or timeouts and act accordingly.
+- There is no limit to the duration of a Workflow.
 
 ### Workflow Execution
 
-An instance of a _Workflow_. The instance can be in the process of executing
-or it could have already completed execution.
+An instance of a [Workflow](#workflow).
+
+- The instance can be in the process of executing or it could have already completed execution.
```

Comment:
A WorkflowExecution can be comprised from multiple Workflow Runs. When a workflow history grows too large it can call [Continue as New]() to create a new run atomically.